src/main/java/org/wecancodeit/pantryplus/Cart.java

```diff
@@ -30,6 +30,10 @@ public long getId() {
 
 	public LineItem getLineItemByProductId(long productId) {
 		for (LineItem lineItem : lineItems) {
+			if (lineItem instanceof CountedLineItem) {
```

Comment:
Not sure this check is needed, the method getLineItemByProductId isn't related to the quantity field found in CountedLineItem and the ultimate result of this if statement is the same as the previous one.